bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Tally/CiphertextTally.cs

```diff
@@ -7,11 +7,7 @@
 
 namespace ElectionGuard.Decryption.Tally;
 
-/// <summary>
-/// The encrypted representation of all contests in the election.
-/// A `CiphertextTally` accepts cast and spoiled ballots and accumulates a tally on the cast ballots.
-/// </summary>
-public record CiphertextTally : DisposableRecordBase, IEquatable<CiphertextTally>
+public record CiphertextTallyRecord : DisposableRecordBase
```

Comment:
broke this out into a base class. it has the same name as one in the ui project for now but we should consolidate these objects at some opint

Reply:
You could have also just used a different name here to save on the duplicate when we knew about it to start with.